solutions/standard/DA-types.md

```diff
@@ -0,0 +1,51 @@
+# Configuring complex inputs for KMS in IBM Cloud projects
+
+Several optional input variables in the IBM Cloud [KMS all inclusive deployable architecture](https://cloud.ibm.com/catalog#deployable_architecture) use complex object types. You specify these inputs when you configure deployable architecture.
+
+* Rules For Context-Based Restrictions (`cbr_rules`)
+
+
+## Rules For Context-Based Restrictions <a name="cbr_rules"></a>
+
+The `cbr_rules` input variable allows you to provide a list of rules for the target service in this case the KMS all inclusive DA instance to enforce access restrictions for the instance based on the context of access requests. Context are criterias that include the network location of access requests, the endpoint type from where the request is sent etc ..
+
+- Variable name: `cbr_rules`.
+- Type: A list of objects. Each object represents a set of rules
+- Default value: An empty list (`[]`).
+
+### Options for cbr_rules
+
+  - `description` (required): The description of the rule to create.
+  - `account_id` (required): The IBM Cloud Account ID
+  - `rule_contexts` (required): (List) The contexts the rule applies to
+      - `attributes` (optional): (List) Individual context attributes
+        - `name` (required): The attribute name.
+        - `value`(required): The attribute value.
+
+  - `enforcement_mode` (required): The rule enforcement mode: `enabled` - The restrictions are enforced and reported. This is the default. `disabled` - The restrictions are disabled. Nothing is enforced or reported. `report` - The restrictions are evaluated and reported, but not enforced.
```

Comment:
Please make the `enforcement_mode` type as nested bullet points, this one will be confusing for the users.

Reply:
fixed https://github.com/terraform-ibm-modules/terraform-ibm-kms-all-inclusive/commit/806305d97382a08396e3ba9111e5efdbda2cf2cc